Dockerfile

```diff
@@ -1,5 +1,20 @@
+FROM golang:1.9 as builder
+
+ARG PACKAGE=github.com/projectriff/shell-function-invoker
+ARG COMMAND=cmd/shell-function-invoker
```

Comment:
could you make `COMMAND` be just `shell-function-invoker`?

Reply:
Done.